documentation/meta/monitoring/runbooks/api_http_5xx_above_threshold.md

```diff
@@ -1,12 +1,12 @@
 # Run Book: API Production HTTP 5XX responses count above threshold
 
 ```{admonition} Metadata
-Status: **Unstable**
+Status: **stable**
 
-Maintainer: @krysaldb
-
-Alarm link:
-- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/API+Production+2XX+responses+count+above+threshold?>
+Alarm links:
+- <https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#alarmsV2:alarm/API+Production+HTTP+5XX+responses+count+over+threshold?>
+- [ECS-Production-Dashboard](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#dashboards/dashboard/ECS-Production-Dashboard)
+- [Production Database + Elasticsearch dashboard](https://us-east-1.console.aws.amazon.com/cloudwatch/home?region=us-east-1#dashboards/dashboard/Service-Overview)
 ```
 
 ## Severity Guide
```

Comment:
It might be helpful to add some sample Cloudwatch Log Insights queries here to make it easier for folks to find the source of the 500s, namely:

```
fields request, @timestamp, @message
| filter status >= 500
| sort @timestamp desc
```

And perhaps a link to the log insights query page for this log group.

Reply:
Great suggestion! First I thought the query might be too much of documentation specific related to CloudWatch but it's short enough that seems worth it. Thank you! Changes applied.